model-optimizer/extensions/middle/quantize_linear_resolver.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.middle.MulFakeQuantizeFuse import MulFakeQuantizeFuse
+from extensions.middle.ReluQuantizeFuse import ReluFakeQuantizeMark
+from extensions.ops.Cast import Cast
+from extensions.ops.elementwise import Mul
+from extensions.ops.fakequantize import FakeQuantize
+from mo.front.common.partial_infer.utils import float_array, int64_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.const import Const
+from mo.ops.reshape import Reshape
+from mo.utils.error import Error
+
+
+class QuantizeLinearResolver(MiddleReplacementPattern):
+    """
+    Replaces QuantizeLinear with FakeQuantize
+    """
+    enabled = True
+    force_clean_up = True
```

Comment:
Force clean up is an expensive operation. Why do you need it after this transformation?